pygotham/talks/models.py

```diff
@@ -90,7 +143,10 @@ class Talk(db.Model):
     )
 
     user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
```

Comment:
That same applies to `recording_release`, although that should be replaced by a property.